samples/nrf9160/gnss/Kconfig

```diff
@@ -89,6 +120,30 @@ config GNSS_SAMPLE_SUPL_PORT
 
 endif # GNSS_SAMPLE_ASSISTANCE_SUPL
 
+if GNSS_SAMPLE_MODE_CONTINUOUS
+
+choice
+	default GNSS_SAMPLE_POWER_SAVING_DISABLED
+	prompt "Select GNSS power saving mode (duty-cycling)"
+
+config GNSS_SAMPLE_POWER_SAVING_DISABLED
+	bool "No power saving"
+
+config GNSS_SAMPLE_POWER_SAVING_PERFORMANCE
+	bool "Power saving without significant performance degradation"
+
+config GNSS_SAMPLE_POWER_SAVING_POWER
```

Comment:
Maybe you could use GNSS_SAMPLE_POWER_SAVING_BATTERY to avoid repeating POWER twice. Other options I came up with:
-GNSS_SAMPLE_POWER_SAVING_DISABLED/ENABLED/AGGRESSIVE
-GNSS_SAMPLE_POWER_SAVING_DISABLED/MODERATE/HIGH

Reply:
DISABLED/MODERATE/HIGH sounds good, I'll go with that.